pkg/apis/agones/v1/gameserver.go

```diff
@@ -135,6 +135,8 @@ type GameServerSpec struct {
 	Health Health `json:"health,omitempty"`
 	// Scheduling strategy. Defaults to "Packed".
 	Scheduling apis.SchedulingStrategy `json:"scheduling,omitempty"`
+	// LogLevel specifies the log level for gameserver sidecar.
+	LogLevel string `json:"loglevel,omitempty"`
```

Comment:
Random thought: Is "loglevel" too generic? Considering it's just for the sdk sidecar - should we pick a name that's more specific? `sdk-server-loglevel` (seems long), not sure what the right answer is here,  figured I would ask the question first.

Just wondering if the name might be confusing.

Reply:
yes, you are right, that would suit better.